pkg/kncloudevents/event_dispatcher_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package channel
+package kncloudevents_test
```

Comment:
package `_test` ?

Reply:
yes, to prevent an import cycle. We are importing here `pkg/eventingtls/eventingtlstesting` for the TLS tests, which is using `pkg/kncloudevents`.

Reply:
meh - ok